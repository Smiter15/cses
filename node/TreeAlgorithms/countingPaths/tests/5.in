10 10
1 3
1 8
5 1
2 8
9 4
1 4
10 6
5 7
6 8
10 6
8 10
1 10
3 2
8 4
6 10
9 5
1 5
10 5
10 8
