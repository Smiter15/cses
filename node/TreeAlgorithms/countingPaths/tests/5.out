6 1 1 2 3 6 0 6 1 6 
